Gemfile

```diff
@@ -17,7 +17,7 @@ group :test do
   gem 'rspec'
   gem 'beaker', *location_for(ENV['BEAKER_VERSION'] || '~> 2.9.0')
   if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
-    gem 'sqa-utils', '~> 0.11'
+    gem 'sqa-utils', '0.12.1'
```

Comment:
Well if you don't mind dealing with the churn of updating the Gemfile again after this on both master and 2.1.x branches, I recommend doing that...